docs/experiments.rst

```diff
@@ -0,0 +1,318 @@
+.. _Experiments:
+
+Experiments
+===========
+
+You might want to take a look at the following pages before exploring experiments:
+
+- :ref:`QuickStart`
+- :ref:`AtVariables`
+- :ref:`Instances`
+- :ref:`Builds`
+- :ref:`Revisions`
+- :ref:`Variants`
+
+On this page we describe how to specify experiments in our ``experiments.yml`` file. We will see how
+to use builds in experiments and enable options such as repeating experiments or setting a timer.
+Moreover, we will see how to modify the ``sbatch`` command when using simexpal with Slurm and how to
+set environment variables.
```

Comment:
State this in a second paragraph. Slurm needs more introduction (that clears up that simexpal can be used together with batch scheduler -- not only Slurm). Not everybody will know what Slurm is.

Reply:
I stated that simexpal can be used with batch schedulers and stated Slurm and SGE as examples. (Although SGE is not really mentioned in the documentation at this point. I will add it later at some point.)